### ISSUE TITLE
Feature Request: Wiki Search

### ISSUE THREAD

**Issue by Davst** [xKynn/zana#18]
Would be handy if Zana could search the wiki.

**Comment by xKynn** [comment]
Could you describe this in further detail? Search the wiki in what capacity?

**Comment by Davst** [comment]
Ah yeah, was just considering that I'd be nice to get a search of the wiki, similarly presented with the 5 resutls type thing that you get when you misspelled an item or so.

Useful would be if you are looking for generalized pages liek the one on way.. Resistance (https://pathofexile.fandom.com/wiki/Resistance) or if you are looking for one of the meta pages like all sources of x or such that don't really fit into the current searches.

Considering the Resistance page example, a wiki hardlink wouldn't be terrible either like !wiki Resistance and return the url link

**Comment by xKynn** [comment]
Fair enough I can definitely see that being useful. Consider it added to the current to-do list :+1:
